makeframemd5

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+SCRIPTDIR=$(dirname "${0}")
+. "${SCRIPTDIR}/mmfunctions" || { echo "Missing '${SCRIPTDIR}/mmfunctions'. Exiting." ; exit 1 ;};
+
+_usage(){
+    echo
+    echo "$(basename "${0}") ${VERSION}"
+    echo "Produces framemd5s for video files in a target directory."
+    echo "Usage: $(basename "${0}") [ -h ] package1 [ package2 ... ]"
+    echo "  -h (show usage)"
+    exit
+}
+[ "${#}" = 0 ] && _usage
+
+OPTIND=1
+while getopts ":h" OPT ; do
+    case "${OPT}" in
+        h) _usage ;;
+        *) echo "bad option -${OPTARG}" ; _usage ;;
+        :) echo "Option -${OPTARG} requires an argument" ; _writeerrorlog "makeframemd5" "The option selected required an argument and none was provided. The script had to exit." ; exit 1 ;;
```

Comment:
hmm good point! I was recycling that from another mm script. I'll just get rid of that option for now